curvefs/src/client/fuse_client.cpp

```diff
@@ -634,6 +642,13 @@ CURVEFS_ERROR FuseClient::FuseOpReadDir(fuse_req_t req, fuse_ino_t ino,
         *buffer = nullptr;
         *rSize = 0;
     }
+
+    VLOG(1) << "batch get inode size = " << inodeIds.size();
+    auto rt = inodeManager_->BatchGetInodeAsync(&inodeIds);
```

Comment:
Use another attr cache to avoid the corruption of inode lru cache， and use BatchGetInodeAttr also.